llvm/lib/Target/X86/X86TargetMachine.cpp

```diff
@@ -105,6 +105,7 @@ extern "C" LLVM_EXTERNAL_VISIBILITY void LLVMInitializeX86Target() {
   initializeX86ReturnThunksPass(PR);
   initializeX86DAGToDAGISelPass(PR);
   initializeX86ArgumentStackSlotPassPass(PR);
+  initializeAtikinPass(PR);
```

Comment:
Please build a pass as dynamic lib to reduce impact on common files.

Reply:
How to do it? I tried to do it in a separate folder, but nothing worked. It just didn’t register my pass.

Reply:
You may look into #100 as example.

Reply:
ready